.changeset/neat-camels-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/coinmarketcap-adapter': patch
+---
+
+Add CID for stETH on coinmarketcap
```

Comment:
Seems like this changeset should have been consumed first, no?

Reply:
Oooh, it's probably because of the auto-update of `develop` because the PR of this branch to `develop` was approved but not merged.

It's likely you'll have to reset this branch up to commit [a1a0eca](https://github.com/smartcontractkit/external-adapters-js/pull/1839/commits/a1a0eca34083ce03e466eee3e166ab334aa35050)

Reply:
Sure, let me try that

Reply:
Oops, looks like we need to go one more commit back. Changeset still included.
Probably gave you the wrong commit sha above, apologies for that.

Reply:
No worries, I should've double-checked it too. Fixed now